1037-treeview-design-guidance.md

```diff
@@ -0,0 +1,33 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-17
+updated: 2024-10-22
+---
+
+# Tree View Design Guidance
+
+The [tree view](https://clarity.design/documentation/tree-view) component presents a hierarchical list of items that have a parent-child relationship where children can be toggled into view by expanding or collapsing their parent item.
+
+## Guidance
+
+- **Must** use a tree view to display and work with large amounts of hierarchical information.
+- **Must not** use too many levels unless it is for user-generated information. Instead, consolidate the layers if possible.
+- **Should not** use a tree view for only one level of nested information. Use an [accordion](https://clarity.design/documentation/accordion) instead.
+
+## Icon Tree
+
+- **Must** maintain consistency for icons. If icons need to be added to a tree view, apply the icons on all items. **Must not** apply icons to only some items in a tree view.
```

Comment:
```suggestion
- Design teams **must** maintain consistency in icon usage within a tree view. If icons are added, they should be applied to all items to ensure a cohesive experience. Design teams **must not** apply icons to only some items in a tree view, as this can create confusion and disrupt the visual hierarchy.
```